companion/src/eeprominterface.cpp

```diff
@@ -1843,3 +1844,16 @@ SimulatorInterface *GetCurrentFirmwareSimulator()
   else
     return NULL;
 }
+
+void FlightModeData::setDefaultLinkedFlightModes(const int phase)
```

Comment:
Considered overloading clear() to maintain current functionality and support new.
Checked a few of the other class clear() functions and the norm was for clear() to zeroise the memory space only. Any other defaults were handled by other functions.
Also I am aware that the whole area of global variables is being reworked in 2.2 so wanted to keep this fix code somewhat isolated.
As a novice I took the safe route with the change so the main devs can make a call. Happy to rework into clear() overloaded or not. Please advise.